Source/WebKit/WebProcess/GPU/webrtc/MediaRecorderPrivate.h

```diff
@@ -71,8 +66,35 @@ class MediaRecorderPrivate final
     void pauseRecording(CompletionHandler<void()>&&) final;
     void resumeRecording(CompletionHandler<void()>&&) final;
 
-    // GPUProcessConnection::Client
-    void gpuProcessConnectionDidClose(GPUProcessConnection&) final;
+    void gpuProcessConnectionDidClose();
+
+    class GPUProcessDidCloseObserver final
```

Comment:
This should just be a class declaration in the header.  The entirety of the class can be moved to the cpp.

header:
class GPUProcessConnectionCloseObserver;
Ref< GPUProcessConnectionCloseObserver > m_gpuProcessDidCloseObserver;

cpp:
class MediaRecorderPrivate::GPUProcessConnectionCloseObserver {
...
};

Reply:
Done